v2/constants.js

```diff
@@ -1,3 +1,29 @@
 export const routes = {
-  home: '/'
+  home: '/',
+
+  /*
+   * Opportunities
+   */
+
+  /* public */
+  OPPORTUNITIES: '/opportunities',
+  OPPORTUNITY: '/opportunities/[id]',
```

Comment:
Nice, but by this logic I assume `home` should be `HOME` since it has no subroutes?

```suggestion
  HOME: '/',

  /*
   * Opportunities
   */

  /* public */
  OPPORTUNITIES: '/opportunities',
  OPPORTUNITY: '/opportunities/[id]',
```

Reply:
Indeed, will fix!